io_ogre/config.py

```diff
@@ -25,42 +26,63 @@
 CONFIG_FILEPATH = os.path.join(CONFIG_PATH, CONFIG_FILENAME)
 
 _CONFIG_DEFAULTS_ALL = {
-    'MESH' : True,
-    'SCENE' : True,
-    'COPY_SHADER_PROGRAMS' : True,
-    'MAX_TEXTURE_SIZE' : 4096,
+    # General
     'SWAP_AXIS' : 'xyz', # ogre standard is 'xz-y', but swapping is currently broken
-    'SEP_MATS' : True,
-    'SELONLY' : True,
+    'MESH_TOOL_EXPORT_VERSION' : 'v2',
+    
+    # Scene
+    'SCENE' : True,
+    'SELECTED_ONLY' : True,
     'EXPORT_HIDDEN' : True,
+    'EXPORT_USER' : True,
```

Comment:
are you trying to sneak that one back in?

Reply:
LOL, it was a mistake. :-D

But I can't pass your eagle eyes.

Reply:
I left the `EXPORT_USER` code commented for future sneakiness...

Just kidding, I want to see how to avoid leaving an empty "<userData />" in the .scene exported file.

Besides, with some objects I get something like this:
      <userData >
        <property data='&lt;bpy id prop: owner="OBCube", name="cycles_visibility", address=0x000002664D72E308&gt;' name="cycles_visibility" type="IDPropertyGroup" />
        <property data='&lt;bpy id prop: owner="OBCube", name="cycles", address=0x000002664D72DE88&gt;' name="cycles" type="IDPropertyGroup" />
      </userData>

Which is ugly as heck and don't know where this comes from.

Reply:
maybe some third-party addon? https://blender.stackexchange.com/questions/140006/edit-custom-properties-cycles-visibility